src/page-parole/PageParole.js

```diff
@@ -46,6 +47,9 @@ const SECTIONS = [
     vizData: {
       districtOffices: paroleDistrictOffices,
       successByMonth: supervisionSuccessByMonth.filter(recordIsParole),
+      successByDemographics: supervisionSuccessDemographics
+        .filter(recordIsParole)
+        .filter((record) => record.metric_period_months === "36"),
```

Comment:
@jjdressel11 @hobuobi the source file has metric periods ranging from 1 to 36 months; I chose this to match what we are showing in the timeline. 

This filter is discarding like 80% of the data in this file! Is this an opportunity to either a) show people more data or b) reduce the amount of data we are sending over the wire? This data file is by far the largest one we have seen so far.

Reply:
Yes I will update on the query side!